src/accelerate/utils/other.py

```diff
@@ -31,6 +32,13 @@
     import torch_xla.core.xla_model as xm
 
 
+def is_compiled_module(module):
+    """Check whether the module was compiled with torch.compile()"""
```

Comment:
Mild nit, space out the `"""` to be under
```suggestion
    """
    Check whether the module was compiled with torch.compile()
    """
```